lang/en.yml

```diff
@@ -3,18 +3,22 @@ en:
     ASSERTLOGINBUTTON: 'Share your details with {orgname}'
     AUTHENTICATOR_NAME: 'RealMe Account'
     LOGINBUTTON: Login
+  SilverStripe\RealMe\Authenticator\LoginHandler:
+    LOGINFAILURE: 'Unfortunately we''re not able to log you in through RealMe right now. Please try again shortly.'
```

Comment:
```suggestion
    LOGINFAILURE: "Unfortunately we're not able to log you in through RealMe right now. Please try again shortly."
```

Reply:
Good spot.  There's actually lots of strings in en.yml that had this weird method of single quoting, so I've converted them all to be surrounded with double quotes